cpp-client/deephaven/client/include/private/deephaven/client/server/server.h

```diff
@@ -97,7 +98,9 @@ class Server : public std::enable_shared_from_this<Server> {
   typedef SFCallback<ExportedTableCreationResponse> EtcCallback;
 
 public:
-  static std::shared_ptr<Server> createFromTarget(const std::string &target, const std::string &authorizationValue);
+  static std::shared_ptr<Server> createFromTarget(
+      const std::string &target,
+      const deephaven::client::ClientOptions &client_options);
```

Comment:
I might add
```
typedef deephaven::client::ClientOptions ClientOptions;
```

at line 94 or so, so that this (and below) can be shortened to `ClientOptions`